frb_example/integrate_third_party/rust/src/api/override_web_audio_api.rs

```diff
@@ -138,6 +158,47 @@ pub impl Event {
     }
 }
 
+#[ext]
+pub impl ScriptProcessorNode
+{
+    // NOTE: The original name was `set_onaudioprocess` and here the new name has `_`
+    fn set_on_audio_process
+    (
+        &self,
+        callback: impl Fn(AudioProcessingEvent) -> DartFnFuture<()> + Send + 'static + std::marker::Sync,
+    )
+    {
+        let callback = Arc::new(callback);
+        self.set_onaudioprocess(move |event|
+        {
+            let callback_cloned = callback.clone();
+            FLUTTER_RUST_BRIDGE_HANDLER
+                .async_runtime()
+                .spawn(async move { callback_cloned(event).await });
+        })
+    }
+
+    fn frb_override_connect(&self, dest: &dyn AudioNode) {
+        self.connect(dest);
+    }
+}
+
+#[ext]
+pub impl AudioBufferSourceNode
+{
+    fn frb_override_connect(&self, dest: &dyn AudioNode)
+    {
+        self.connect(dest);
+    }
```

Comment:
btw wondering is this needed - seems handle_audio_node_trait_impls_override already does so, thus maybe we can safely remove it